### PR TITLE
update Shopware-Font

### PR DESCRIPTION
**Pull request by tinect**
### 1. Why is this change necessary?
- adding new icons: instagram, xing, youtube + circle-versions
- remove font-type .eot cause it's just for internet explorer which supports woff, too

### 2. What does this change do, exactly?
- used same font as Shopware used for the other ones: Entypo+
- change shopware's instagram-icon to instagram3, cause implementing new one including circle-version
- correct link to styletile in icons.less
- fonts were generated by IcoMoon resulting a smaller size for woff and ttf: 69KB => 51KB
- adds woff2 font-type which is much smaller than gzipped woff

icons

### 5. Which documentation changes (if any) need to be made because of this PR?
styletile

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

**Comment by mitelg** [comment]
replaces #1540 
Ticket Number is https://issues.shopware.com/issues/SW-21285

**Comment by xf-** [comment]
Optional grunt plugin would be great for easy adjustments
https://www.npmjs.com/package/grunt-icomoon of the icon set

File: .htaccess [hunk 1]
```diff
@@ -64,7 +64,7 @@ DirectoryIndex shopware.php
     AddOutputFilterByType DEFLATE text/html text/xml text/plain text/css text/javascript application/javascript application/json application/font-woff application/font-woff2 image/svg+xml
 </IfModule>
 
-<Files ~ "\.(jpe?g|png|gif|css|js|woff|eot)$">
+<Files ~ "\.(jpe?g|png|gif|css|js|woff|woff2)$">
```
Comment (xf-) [inline .htaccess]: use same syntax like jpeg jpg with a question-mark woff?

**Comment by wolv-dev** [comment]
@tinect Can you please replace the instagram icon with the new one?

**Comment by tinect** [comment]
> @tinect Can you please replace the instagram icon with the new one?

Thank you, @wolv-dev  :-)

**Comment by mitelg** [comment]
thanks for your contribution @tinect :+1: :blue_heart: 
will be merged soon :wink:

**Pull request closed without merging**